versioned_docs/version-6.x/params.md

```diff
@@ -65,7 +65,7 @@ function DetailsScreen({ route, navigation }) {
 ```
 
 <video playsInline autoPlay muted loop>
-  <source src="/assets/navigators/passing-params.mp4" />
+  <source src="/assets/7.x/fundamentals/navigatePassParams.mp4" />
```

Comment:
I believe this isn't the same example, as on the previous recording 🤔 Is it intentional?

Reply:
You are right 👀 I updated the wrong file